README.md

```diff
@@ -28,51 +28,52 @@ npx @temporalio/create@latest my-project-name
 
 ### Basic
 
-- [**Basic hello world**](./hello-world): Simple example of a Workflow Definition and an Activity Definition.
-  - Variant: [Basic hello world with mTLS](./hello-world-mtls) shows how to connect to your Temporal Cloud namespace with mTLS authentication.
-- [**Pure ES Modules**](./fetch-esm): Configure Temporal with TypeScript and Pure ESM.
+- [**Basic hello world**](https://github.com/temporalio/samples-typescript/tree/main/hello-world): Simple example of a Workflow Definition and an Activity Definition.
```

Comment:
What's the reason for making these absolute URLs?

Reply:
swyx wanted them clickable in case people clone the whole repo and are reading it locally